cmd/siac/hostcmd.go

```diff
@@ -283,6 +283,17 @@ RPC Stats:
 			currencyUnits(totalRevenue))
 	}
 
+	// if wallet is locked print warning
+	walletstatus := new(api.WalletGET)
+	walleterr := getAPI("/wallet", walletstatus)
+	if err != nil {
+		die("Could not get wallet status. A working wallet is needed in order to operate your host. Error:", walleterr)
+	} else {
+		if !walletstatus.Unlocked {
+			fmt.Println("\nWARNING:\n	In order to operate your host you need to unlock your wallet using the command \"siac wallet unlock\"")
+		}
+	}
```

Comment:
you can save some lines here by putting `else if` on the same line. Also, I don't think it's necessary to specify the exact command. How about `"\nWarning: Your wallet is locked. You must unlock your wallet for the host to function properly."`